include/os/linux/zfs/sys/zfs_ioctl_os.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _LINUX_ZFS_IOCTL_OS_H_
+#define	_LINUX_ZFS_IOCTL_OS_H_
+
+#define	ZONED "zoned"
+
+#endif
```

Comment:
Does Linux use the convention where the endif says what ifdef it is ending?

Reply:
Yes, usually the `#endif` gets commented if it's at unclear what the preceding `#if` is.  Which means most of the time in practice.